examples/with-redux-wrapper/pages/_app.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import withRedux from 'next-redux-wrapper'
+import {initStore} from '../store'
+
+export default withRedux(initStore)(class MyApp extends React.Component {
```

Comment:
It's mandatory you follow the *exact* example of the readme. So extend from `App` and implement `render` with the `Container`

Reply:
@timneutkens are you OK with provided explanation?